packages/exoflex/src/components/Drawer/DrawerSection.tsx

```diff
@@ -38,18 +61,30 @@ export default function DrawerSection(props: Props) {
       );
     }
     return (
-      <View style={styles.headerFullWrapper} testID="drawerHeaderFullImage">
+      <View
+        accessibilityRole="header"
+        style={styles.headerFullWrapper}
+        testID="drawerHeaderFullImage"
+      >
         <Image source={headerSource} style={styles.headerFullImage} />
       </View>
     );
   };
 
   return (
-    <View style={styles.container}>
+    <View
+      {...otherProps}
+      accessibilityRole={accessibilityRole || 'menubar'}
+      style={styles.container}
+    >
       <Header />
       <View style={{ flex: 1 }}>{children}</View>
       {footerLabel && (
         <DrawerItem
+          accessibilityRole={
+            footerAccessibilityRole || defaultFooterAccessibilityRole
+          }
+          accessibilityState={footerAccessibilityState}
```

Comment:
Anyway, I don't think we should remove `footerAccessibilityLabel` and `footerAccessibilityHint`. Instead, we should properly pass them on to this component along with other `footerAccesibility`-ies.

Reply:
Oops, missed that one.